src/uct/ib/base/ib_iface.c

```diff
@@ -1534,6 +1582,16 @@ ucs_status_t uct_ib_iface_query(uct_ib_iface_t *iface, size_t xport_hdr_len,
     iface_attr->latency.c += numa_latency;
     iface_attr->latency.m  = 0;
 
+#if HAVE_CUDA
```

Comment:
why do we need HAVE_CUDA macros?

Reply:
Sorry. If cuda device isn't detected we just return cuda_latency = 0. I'll change this now.